python/src/tests/tasks/test_differential_expression.py

```diff
@@ -296,20 +307,20 @@ def test_rest_keyword_only_adds_cells_in_the_same_hierarchy(self, mock_dynamo_ge
 
 
 """
-    def test_dynamodb_call_is_made_once_when_vs_rest(self, mock_dynamo_get):
-        m, dynamodb = mock_dynamo_get
+    def test_dynamodb_call_is_made_once_when_vs_rest(self, mock_S3_get):
```

Comment:
What do we want to test here now?

Reply:
The changes I did here were accidental (a replace all) but I left them because they were commented tests anyway (the `"""`).

For consistency though I should probably leave them as the were before, with `mock_dynamo_get`.

Reply:
Oops, did not notice that the test was commented out :-)